include/geode/inspector/topology/brep_topology.h

```diff
@@ -26,19 +26,31 @@
 #include <geode/basic/pimpl.h>
 
 #include <geode/inspector/common.h>
+#include <geode/inspector/information.h>
 #include <geode/inspector/topology/brep_blocks_topology.h>
 #include <geode/inspector/topology/brep_corners_topology.h>
 #include <geode/inspector/topology/brep_lines_topology.h>
 #include <geode/inspector/topology/brep_surfaces_topology.h>
 
 namespace geode
 {
-    struct ComponentMeshVertex;
     class BRep;
 } // namespace geode
 
 namespace geode
 {
+    struct opengeode_inspector_inspector_api BRepTopologyInspectionResult
```

Comment:
```suggestion
    struct BRepTopologyInspectionResult
```
No need if nothing is implemented in the cpp file

Reply:
Do the same for other struct